test/models/release_test.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 require_relative '../test_helper'
 
-SingleCov.covered! uncovered: 2
+SingleCov.covered! uncovered: 3
```

Comment:
add test for github_status ?

Reply:
It doesn't contain any logic, and is very unlikely to break. I think a test is superfluous, unless you're adamant about it.

Reply:
The controller test already exercises the code path.

Reply:
hmmm I'd still add the test since we are trying to keep everything fully unit-tested.